src/plugins/intel_npu/tools/protopipe/src/simulation/ov_layers_reader.cpp

```diff
@@ -128,6 +128,18 @@ static void cfgOutputPostproc(ov::preprocess::PrePostProcessor& ppp, const std::
     }
 }
 
+static void cfgPartialShapes(const std::shared_ptr<ov::Model>& model, 
+                             const AttrMap<std::vector<uint64_t>> shape_map) {
+
+    std::map<std::string, ov::PartialShape> partial_shapes;
+    for (const auto& shape : shape_map) {
+        std::vector<ov::Dimension> vecDimension;
+        for (const auto& dimension : shape.second) { vecDimension.emplace_back(dimension); }
```

Comment:
can it be just?
```
for (const auto& [layer_name, shape] : shape_map) {
    partial_shape.emplace(layer_name, shape);
}
```

Reply:
I will check to see if this works. I am not sure and would be a bit surprised if it does work.

Reply:
Working done